spec/services/hyrax/custom_queries/find_collections_by_type_spec.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
-RSpec.describe Hyrax::CustomQueries::FindCollectionsByType, valkyrie_adapter: :test_adapter do
+RSpec.describe Hyrax::CustomQueries::FindCollectionsByType, valkyrie_adapter: :test_adapter, skip: !Hyrax.config.use_valkyrie? || !Hyrax.config.disable_wings do
```

Comment:
`Hyrax::CustomQueries::FindCollectionsByType` is only relevant for Koppie, based on the following:

Applications like Dassie rely on an `ActiveFedora` query as shown in `#collections` from `app/models/hyrax/collection_type.rb`:

```ruby
##
    # @return [Enumerable<Collection, PcdmCollection>]
    def collections(use_valkyrie: Hyrax.config.use_valkyrie?)
      return [] unless id
      return Hyrax.custom_queries.find_collections_by_type(global_id: to_global_id.to_s) if use_valkyrie
      ActiveFedora::Base.where(Hyrax.config.collection_type_index_field.to_sym => to_global_id.to_s)
    end
```

Applications that utilize Wings rely on `Wings::CustomQueries::FindCollectionsByType` instead of `Hyrax::CustomQueries::FindCollectionsByType` to lookup collections by collection type.

Therefore, I am skipping these tests when the condition `!Hyrax.config.use_valkyrie? || !Hyrax.config.disable_wings` is met.